src/main/java/uk/ac/ebi/embl/api/validation/check/file/TSVFileValidationCheck.java

```diff
@@ -52,8 +53,12 @@ public ValidationResult check(SubmissionFile submissionFile) throws ValidationEn
 		try (PrintWriter fixedFileWriter=getFixedFileWriter(submissionFile)) {
              clearReportFile(getReportFile(submissionFile));
 			String templateId = getTemplateIdFromTsvFile(submissionFile.getFile());
-			if(!options.isWebinCLI && StringUtils.isBlank(templateId ) ) {
-				 templateId = new EraproDAOUtilsImpl(options.eraproConnection.get()).getTemplateId(options.analysisId.get());
+			if(!options.isWebinCLI) {
+				if (StringUtils.isBlank(templateId)) {
```

Comment:
The only place the the template id can be is the TSC file. Throw an exception if we can't retrieve it from the TSV file and do not attempt to read it from the database.

So, this code becomes -> 
```
if(templateId == null) {
				throw new ValidationEngineException("Missing template id", ValidationEngineException.ReportErrorType.VALIDATION_ERROR);
}

if(!options.isWebinCLI) {
// Save template ID in the database.
WebinERAService.getWebinERAService(options.getWebinERAServiceUrl(), options.getWebinERAServiceUser(), options.getWebinERAServicePassword()).saveTemplateId(options.analysisId.get(), templateId);
}
```